src/agent/rustjail/src/container.rs

```diff
@@ -25,6 +25,8 @@ use crate::cgroups::mock::Manager as FsManager;
 use crate::cgroups::Manager;
 use crate::log_child;
 use crate::process::Process;
+#[cfg(feature = "seccomp")]
+use crate::seccomp;
```

Comment:
If this feature is optional, we should specify whether it is available in `announce()` (for the logs). We already have a seccomp field in the `GetGuestDetails()` ttRPC response so in fact it might be worth just adding the output of `get_agent_details()` to the output of `announce()`.